zerver/tests/test_send_email.py

```diff
@@ -47,28 +31,41 @@ def test_build_SES_compatible_From_field_limit(self) -> None:
             mail.extra_headers["From"], "{} <{}>".format(limit_length_name, FromAddress.NOREPLY)
         )
 
-    def test_build_SES_incompatible_From_field(self) -> None:
+    def test_build_and_send_SES_incompatible_From_address(self) -> None:
         hamlet = self.example_user("hamlet")
+        from_name = "Zulip"
+        # Address by itself is > 320 bytes even without the name. Should trigger exception.
+        overly_long_address = "a" * 311 + "@zulip.com"
```

Comment:
Might be clearer to make this:
```suggestion
        overly_long_address = "a" * 320 + "@zulip.com"
```
...so that it's patently obvious that it's over 320 bytes, so I don't have to count how long `@zulip.com` is to be convinced. :)